### ISSUE TITLE
Step 2: Creating a Stream Instruction Typo

### ISSUE THREAD

**Issue by DimaK415** [tweepy/tweepy#972]
Step 2: Creating a Stream
We need an api to stream. See Authentication Tutorial to learn how to get an api object. Once we have an api and a status listener we can create our stream object:

```
myStreamListener = MyStreamListener()
myStream = tweepy.Stream(auth = api.auth, listener=myStreamListener())
```
line 2 should be `listener=MyStreamListener()`

**Comment by shuuji3** [comment]
Hi, DimaK415

It seems that the line has already fixed in [the latest document](http://docs.tweepy.org/en/latest/streaming_how_to.html) and it remains errors in the stable or v3.5.0 documentation. So, I think you'd better read the latest edition of documentation if you use Tweepy.

**Comment by Harmon758** [comment]
This was already fixed with https://github.com/tweepy/tweepy/commit/8322cf6252db66c2288fdbf6c218f88518320cb9 (#695) and released with Tweepy v3.6.0.
As already explained, you should refer to the latest version when referencing the documentation and opening issues about it.
As for changing the Default version for Read the Docs, see #1134.
